### PR TITLE
Added a CS question in full-quiz.js

### LINKED ISSUE #29: Adding Computer Science Questions
[developerquiz.org](developerquiz.org) currently has over 600 quiz questions. We are looking to expand on the Computer Science questions and we encourage other developers to add their quiz questions to the site.

You can find the complete list of questions below.
https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/src/data/general-cs-quiz.ts

You can add your own questions to the top of that file.
**Please first check to make sure your question doesn't already exist in the file.**

Here is an example of one of the current CS questions.

```
 {
    Question: "Which one of these is NOT a common time complexity equation for BigO notation?",
    Answer: "O(n14)",
    Distractor1: "O(1)",
    Distractor2: "O(log n)",
    Distractor3: "O(n)",
    Explanation: "O(1), O(log n), O(n) are just some of the common time complexity equations for BigO notation.",
    Link: "https://www.youtube.com/watch?v=zg9ih6SVACc&t=867s",
  },
```
For the link field, please make sure to use a freeCodeCamp article, or freeCodeCamp YouTube video.
If you choose to reference a video, please make sure to include the timestamp for the topic covered.

Here is an example of a timestamp video for BigO notation
https://www.youtube.com/watch?v=zg9ih6SVACc&t=867s

You can read more about how to create timestamps in this [helpful article](https://www.lifewire.com/link-to-specific-part-of-youtube-video-1616414#). 

**Please note: Do not close this issue.** We want to keep this issue open for multiple contributors.

Happy contributing!

### PR DESCRIPTION
**Pull request by kalashjain23**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

 #29

### CODE REVIEW

File: src/data/full-quiz.js [hunk 1]
```diff
@@ -1,4 +1,13 @@
 const fullQuiz = [
+  {
+    Question: "Which of the following is correct for arrays?",
+    Answer: "Size of an array can not be changed once intialized",
+    Distractor1: "Insertion and deletion of an element is efficient",
```
Comment (jdwilkin4) [inline src/data/full-quiz.js:5]: I'd like to modify Distractor1's answer. 
Proposed fix: Insertion and deletion of an element has a time complexity of O(1).

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
HI @kalashjain23 ! 

I think your question looks good.
Just had a small suggestion below. 👍

**Comment by kalashjain23** [comment]
Thanks @jdwilkin4 for that suggestion!!😁 I have made the required changes..

**Review by jdwilkin4 — APPROVED** [review]
LGTM! 👍

**Comment by jdwilkin4** [comment]
Hi @kalashjain23 ! 

Thank you for your quiz question! 
We look forward to future contributions 👍

**Pull request merged**